schema/job/compute.json

```diff
@@ -52,7 +53,13 @@
         },
         "arguments": {
             "description": "Optional arguments specific to using application - VASP, Quantum Espresso, etc. Specified elsewhere",
-            "type": "object"
+            "type": "object",
+            "default": {},
+            "allOf": [
```

Comment:
Should we define arguments in `software_directory/modeling/advanced_compute_arguments.json` using `oneOf` instead of `allOf`, right? Logically this would need to be the advanced compute arguments for one of the applications, not all of them at once...